package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>cyclonedds</name>
+    <version>0.1.0</version>
```

Comment:
I am not sure what versioning scheme you are aiming for. Maybe 0.1.0 might be confusing with the same number being advertised on the website being from March / the tag being from February.

Just a note - it can stay as is. Before a new release is made this needs to be bumped and the repo tagged.

Reply:
I agree it should be updated. I pulled this version number from the top-level CMakeLists.txt